### PR TITLE
Add k8s workflows

### PR DESCRIPTION
**Pull request by makarenkoo**
Two workflows: to **deploy** and to **delete** from kubernetes.

Example of use:
```
helm_upgrade_t2_eks:
    uses: coingaming/.github/.github/workflows/deploy-to-k8s.yml@add-k8s-workflows
    with:
      AWS_REGION: "eu-central-1"
      CLUSTER_NAME: "framework-cluster"
      DOCKER_IMAGE: "heathmont/connect_flask:v1.13.0-50e1b74e6"
      HELM_RELEASE_NAME: "connect-flask-release"
      K8S_NAMESPACE: "connect-flask"
      PATH_TO_HELM_CHART: "./kubernetes/connect_chart"
      PATH_TO_HELM_VALUES: "./kubernetes/connect_chart/values.t2.yaml"
      HELM_TIMEOUT: 10
      HELM_DEBUG: false
    secrets:
      AWS_OIDC_ROLE_TO_ASSUME: ${{ secrets.AWS_FRAMEWORK_DEV_OIDC_ROLE }}
      GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

### CODE REVIEW

**Review by altuntasfatih — APPROVED** [review]

**Comment by altuntasfatih** [comment]
thank you,, we can configure it from application pipeline for any env or any cluster so that is good.

File: .github/workflows/deploy-to-k8s.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      - name: Helm Upgrade
+        env:
+          K8S_NAMESPACE: ${{ inputs.K8S_NAMESPACE }}
+          DOCKER_IMAGE: ${{ inputs.DOCKER_IMAGE }}
+          PATH_TO_HELM_CHART: ${{ inputs.PATH_TO_HELM_CHART }}
+          PATH_TO_HELM_VALUES: ${{ inputs.PATH_TO_HELM_VALUES }}
+          HELM_DEBUG: ${{ inputs.HELM_DEBUG && '--debug' || '' }}
+          TIMEOUT: ${{ inputs.HELM_TIMEOUT || inputs.HELM_TIMEOUT.default }}
+        run: |
+          helm upgrade ${K8S_NAMESPACE}-release $PATH_TO_HELM_CHART \
```
Comment (altuntasfatih) [inline .github/workflows/deploy-to-k8s.yml:98]: Lets put namespace before create-namespace
 
```
 helm upgrade ${{env.RELEASE_NAME}} ./kubernetes/admin_chart \
          -f ./kubernetes/admin_chart/values.t2.yaml \
          --namespace ${{env.K8_NAMESPACE}} \
          --create-namespace \
          --set image.name=${{env.IMAGE}} \
          --timeout 10m0s --wait --debug \
          --install
```

**Review by altuntasfatih — APPROVED** [review]

**Review by altuntasfatih — APPROVED** [review]

File: .github/workflows/deploy-to-k8s.yml (hunk 1 above)
Comment (makarenkoo) [reply .github/workflows/deploy-to-k8s.yml]: Changed by your request.

[the remark this answers is not in the document]
Comment (makarenkoo) [reply .github/workflows/deploy-to-k8s.yml]: Introduced a new input HELM_RELEASE_NAME.

**Review by altuntasfatih — APPROVED** [review]

**Pull request merged**